### PR TITLE
add notes based on concepts

### PR DESCRIPTION
**Pull request by myarjunar**
### What does it fix ?
* Ticket : #4069 
* Funded by : DFAT
* Description : Add notes to the bottom of the demographic breakdown table. The notes came from description in concepts.py.

selection_068

### CODE REVIEW

**Comment by timlinux** [comment]
Can you make the concept in bold : **Gender group**: hdakjhsd jkahdjksah hskjahdkjas

**Comment by timlinux** [comment]
LGTM when notes are DRY

**Comment by Charlotte-Morgan** [comment]
@ivanbusthomi - please note changes already made here in your work on #4194

**Comment by Charlotte-Morgan** [comment]
@ivanbusthomi - take a look at this for what has already been implemented 
cf #4134

**Comment by ismailsunni** [comment]
@myarjunar You will need to update this PR with develop, since there is changing variable name. See the travis's error.

File: safe/definitions/field_groups.py [hunk 1]
```diff
@@ -281,3 +282,44 @@
...
+vulnerability_count_group['notes'] += vulnerability_group_notes
+vulnerability_displaced_count_group['notes'] += vulnerability_group_notes
+
+for field_group in all_field_groups:
+    field_group['notes'].insert(
+        0,
+        tr('{group_name} group: {note}'.format(
+            group_name=field_group['name'],
+            note=field_group['description'])))
+    del field_group
```
Comment (ismailsunni) [inline safe/definitions/field_groups.py]: Please add comment here why do we need to do `del field_group`

Comment (myarjunar) [inline safe/definitions/field_groups.py]: done :+1:

**Comment by ivanbusthomi** [comment]
> @ivanbusthomi - take a look at this for what has already been implemented

Yes @Charlotte-Morgan, I really like the idea of having a section that contains many different tables than having many sections that contain a table each. we can then implement the idea of having subtitles for each table within that sections.
like this one
image
and this one
image

**Comment by timlinux** [comment]
Folks I am going to merge this so long we can address ideas in  #4134 in further PR's

Thanks

**Pull request merged**